pages/stable-version.html

```diff
@@ -273,6 +273,7 @@ <h4>Client Tools</h4>
                 <li><a href="https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/Ubuntu2204-Uyuni-Client-Tools/xUbuntu_22.04/">Ubuntu 22.04 (x86_64)</a></li>
                 <li><a href="https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/Debian10-Uyuni-Client-Tools/Debian_10/">Debian 10 (x86_64, aarch64, armv7l, i586)</a></li>
                 <li><a href="https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/Debian11-Uyuni-Client-Tools/Debian_11/">Debian 11 (x86_64, aarch64, armv7l, i586, ppc64le, s390x)</a></li>
+                <li><a href="https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/Debian11-Uyuni-Client-Tools/Debian_12/">Debian 12 (x86_64, aarch64, armv7l, i586, ppc64le, s390x)</a></li>
```

Comment:
```suggestion
                <li><a href="https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/Debian12-Uyuni-Client-Tools/Debian_12/">Debian 12 (x86_64, aarch64, armv7l, i586, ppc64le, s390x)</a></li>
```
I assume the link won't be accessible till release, anyway that 11 seems to be a typo, correct me otherwise.